CHANGELOG.md

```diff
@@ -1,6 +1,71 @@
-## 4.0.0-rc.3
```

Comment:
I was going to ask why this went away, but I guess RC3 never shipped right?

Reply:
yeah, I was originally gonna ship this as RC3 but as I gained confidence in the build I ended up removing the prerelease modifier